_posts/2022-07-18-azure-ad-authentication-cypress.md

```diff
@@ -0,0 +1,185 @@
+---
+layout: post
+font: serif
+title: Azure AD Authentication in Cypress Tests with MSAL
+date: 2022-07-18
+highlight: monokai
+author: Tim Veletta
+tags: azure ad activedirectory cypress authentication msal react
```

Comment:
I notice you didn't specify a Header image here?

Reply:
Yep, will resolve